File: cs50/pset6/cash/cash.py
import cs50
import math

while True:
    owed = cs50.get_float("Change owed: ")
    if owed > 0:
        break
    
cents = round(owed * 100, 0)
quarters = 0
dimes = 0
nickles = 0
pennies = 0
rest = 0

quarters = math.floor(cents / 25)
rest = cents % 25
dimes = math.floor(rest / 10)
rest = rest % 10
nickles = math.floor(rest / 5)
rest = rest % 5
pennies = rest

coins = math.floor(quarters + dimes + nickles + pennies)

print(coins)

File: cs50/pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float owed = -1;
    
    while (owed < 0)
    {
        owed = get_float("Change owed:");
    }
    
    int cents = round(owed * 100);
    
    int quarters = 0;
    int dimes = 0;
    int nickles = 0;
    int pennies = 0;
    int rest = 0;
    
    quarters = cents / 25;
    rest = cents % 25;
    dimes = rest / 10;
    rest = rest % 10;
    nickles = rest / 5;
    rest = rest % 5;
    pennies = rest;
    
    
//    printf("change = %i | quarters = %i | dimes = %i | nickles = %i | pennies = %i | rest = %i\n", cents, quarters, dimes, nickles, pennies, rest);
    
    int coins = quarters + dimes + nickles + pennies;
        
    printf("%i\n", coins);
    
}